api/.env.example

```diff
@@ -0,0 +1,3 @@
+DATABASE_URL=
```

Comment:
Yes! If you use docker-compose up, it will load the .env variables. This is from the [Docker documentation](https://docs.docker.com/compose/environment-variables/):

> You can set default values for environment variables using a [.env file](https://docs.docker.com/compose/env-file/), which Compose automatically looks for in project directory (parent folder of your Compose file). Values set in the shell environment override those set in the .env file.